### ISSUE TITLE
DRAFT - Usability Testing: Combo box improvements

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds#5521]
## Description

Include a high-level description of the issue. What did you expect to happen? What happened instead? What would you like to see changed?

Include any benefits, challenges, or considerations. This can be short and sweet.

## Findings report

<details><summary>Details</summary>
<p>

Pain points felt for 4 out of 5 participants:

- **Doesn’t behave as users expect**. They expect to be able to use 1st letter navigation to get to their selection (“T” takes them to states starting with “T”) - [4 out of 5 participants]. One person said it acted as expected (searched using ‘Texas”) and this user seemed savvy/familiar with combobox functionality conventions.
    - → Should recommendation be for state dropdowns to use the SELECT component instead of the COMBOBOX?
- One person (U4) was surprised by the “x” deselect in the combobox - she usually only sees that in combo boxes where you can make multiple selections - [see clip](https://bixal.dovetailapp.com/data/7GYQdI9ZUR3yo3OuxEG2Oz#:v:h=7oEX7SwL4EKlIZYWp3vULf&s=1)

Positive: One person liked that when you enter something wrong, it gives feedback of “no results found.” Often she sees combo boxes that give no feedback and are just blank.

There is past evidence/research for combobox that bolster the above findings:

- Combobox problems documented in this [combobox bug report](https://github.com/uswds/uswds/issues/5161). Perhaps the recommendation should be to use a SELECT component for state dropdowns, not a combobox.
- [Combobox feedback and recommendations](https://docs.google.com/document/d/11V_0QGD2NCNPLRJkH0gOkCw0kntY9Y_r8U33-hBd-yo/edit?usp=sharing) from research
    - *This document was from before a keyboard behavior enhancement in #5160  - Charlie*
- [Video clip of user struggling with first letter nav issue](https://drive.google.com/file/d/1J25wdVSczuRf7YUfvPpUdt6xH-TLQjvO/view?usp=sharing)

</p>
</details> 

> 💡 **Combo box recommendations:**
>
> Should recommendation be for state dropdowns to use the SELECT component instead of the COMBOBOX? We would need to change guidance if so, since current guidance says not to use select box for more than 15 options.
>
> Also, only offer the deselect “x” in comboboxes where the user can select multiple options.

## Related Issues

- Combo box results →
- Combo box guidance →

### Select vs Combobox

It appears the users are expecting **Select** behavior.

From [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/#:~:text=Printable%20Characters%3A,the%20typed%20characters.):

>Printable Characters:
> - If the combobox is editable, type characters in the combobox. Note that some implementations may regard certain characters as invalid and prevent their input.
> - If the combobox is not editable, optionally moves focus to a value that starts with the typed characters.

In terms of W3C, our components operate as expected, but if users aren't expecting this behavior it could lead to confusion.

Could we add screen reader instructions to enter a _search_ term?

**Comment by mahoneycm** [comment]
Duplicate of #5522
